auth0/data_source_auth0_client_test.go

```diff
@@ -24,24 +24,24 @@ data auth0_client test {
 `
 
 func TestAccDataClientByName(t *testing.T) {
-	rand := random.String(6)
```

Comment:
We won't be able to use the random pkg any more as the state and recorded http interactions need to be the same otherwise, terraform will complain that there's a diff between state tf config. 

We're gonna start using the test name wherever possible as a name.

Random pkg will get removed in a subsequent PR.